software/base/src/test/java/org/apache/brooklyn/tasks/kubectl/ContainerTaskTest.java

```diff
@@ -103,6 +103,7 @@ public void testFailingContainerTask() {
         configBag.put("name", "test-container-task");
         configBag.put("image", "perl");
         configBag.put("commands", commands);
+        configBag.put("timeout", "1m");
```

Comment:
Timeout is a default '1m' in the config key.

Reply:
So... you changes your mind about it yesterday :), got it.